packages/mdc-menu/simple/foundation.js

```diff
@@ -480,11 +549,13 @@ class MDCSimpleMenuFoundation extends MDCFoundation {
     }
 
     this.adapter_.deregisterBodyClickHandler(this.documentClickHandler_);
-    this.adapter_.addClass(MDCSimpleMenuFoundation.cssClasses.ANIMATING);
+    this.adapter_.addClass(MDCSimpleMenuFoundation.cssClasses.ANIMATING_CLOSE);
```

Comment:
Nit: should this be `ANIMATING_CLOSED` (where "open" and "closed" are adjectives as opposed to "open" and "close" being verbs)?

Reply:
ok changed to adj meaning, however, I was thinking those as being verbs in this case. Like in "animating whirl".